src/Microsoft.IO.Redist/src/Microsoft.IO.Redist.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+    <PackageDescription>Downlevel support package for System.IO classes.</PackageDescription>
+    <DefineConstants>$(DefineConstants);MS_IO_REDIST</DefineConstants>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <ClsCompliant>false</ClsCompliant>
+    <Nullable>annotations</Nullable>
+    <IncludePlatformAttributes>true</IncludePlatformAttributes>
+    <DocumentationFile>$(MSBuildThisFileDirectory)Microsoft.IO.Redist.xml</DocumentationFile>
+  </PropertyGroup>
+
+  <!-- Package servicing properties -->
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>6.0.1</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">6.1.0</VersionPrefix>
+    <AssemblyVersion>6.0.0.1</AssemblyVersion>
+    <AssemblyVersion Condition="'$(IsPackable)' == 'true'">6.0.1.0</AssemblyVersion>
```

Comment:
Would it make sense to not set AssemblyVersion and let it be calculated from `VersionPrefix`? This would simplify package versioning.